internal/eval/eval.go

```diff
@@ -2258,7 +2258,7 @@ func evalGETSET(args []string, store *dstore.Store) []byte {
 }
 
 func evalFLUSHDB(args []string, store *dstore.Store) []byte {
-	log.Info(args)
+	slog.Info("FLUSHDB called", slog.Any("args", args))
```

Comment:
eval functions arne't recievers, so they don't have any structs to store loggers in. I can imagine multiple approaches here.

1. use the global logger using slog.Info as we do right now
2. use the logger by passing logger in as a argument to each eval function
3. pass in a `context.Context` that has a logger in it.

I've used the global logger here (since it's just one instance of such a call)

Reply:
That's okay, we are rewriting the eval, which may take care of this.